packages/hadron-build/commands/release.js

```diff
@@ -380,11 +380,14 @@ const installDependencies = (CONFIG, done) => {
     }
     cli.debug('Dependencies installed');
 
-    rebuild({
+    const rebuildOptions = {
       electronVersion: CONFIG.packagerOptions.electronVersion,
       buildPath: path.join(CONFIG.resources, 'app'),
+      onlyModules: ['interruptor', 'kerberos'],
```

Comment:
Is it safe to hardcode modules here? I think electron-rebuild should be able to figure out what needs a rebuild automatically, right?

Reply:
It tries to rebuild also `keytar` and then fails cause our electron version is too old and doesn't have headers. Was looking for something like `excludeModules` but there is nothing like that.

Reply:
Yeah I agree, it would be nice not to hardcode the names, but for now it’s fine I guess – can we add a tech debt ticket to address this?

Reply:
I don't see why not, we could do this also now if someone has an idea, otherwise i can create a ticket.

Reply:
For example we could probably read it from some package json config / or env variable?

Reply:
`hadron-build` takes a lot of options from package.json, so yeah, maybe similar to `asar` options, `rebuild` options can be kept there? This feels a bit cleaner

https://github.com/mongodb-js/compass/blob/fab73e2dfd100f588c1ed4f930186e2f1bdbb064/packages/compass/package.json#L193-L208

https://github.com/mongodb-js/compass/blob/fab73e2dfd100f588c1ed4f930186e2f1bdbb064/packages/hadron-build/commands/release.js#L470

Generally I'm also fine with keeping it as is and having a ticket for the future, just really wasn't sure why exactly we needed to provide them there explicitly

Reply:
Nice, i like that a lot: https://jira.mongodb.org/browse/COMPASS-4889

Made a ticket and referenced this discussion.